src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import FuzzyPathSearch from '../FuzzyPathSearch'
+import { getTypeFromMimeType } from '../../../containers/File'
```

Comment:
Maybe we should move this function somewhere else? Not sure where, but it feels weird to import a utility from a component.

Reply:
I asked myself the same question. And I concluded with this: It's not a component, it's a `File` component's file with a component and tools about its competence. I am not sure this should be move somewhere related to its category rather than related to its responsability/competence.

Reply:
> I am not sure this should be move somewhere related to its category rather than related to its responsability/competence.

I understand, although our code tens to be organized by category. But I don't have a better idea and you've thought about it, so that's ok for me.

Reply:
Totally agree with @enguerran